test/integration/features/basic.feature

```diff
@@ -44,6 +44,15 @@ Feature: Basic
     minishift
     """
 
+  Scenario: Persistent volumes should be listed when minishift start
+    When executing "oc get pv --as system:admin -o=name"
+    Then stderr should be empty
+     And exitcode should equal 0
+     And stdout should contain
+     """
+     persistentvolume/pv0001
+     """
```

Comment:
> However, I don't think you need to deploy an app to make a persistence volume claim. You can claim is directly via the API.

Yes I hope you are talking about https://docs.openshift.com/enterprise/3.1/dev_guide/persistent_volumes.html#requesting-storage I did try it but problem is this should be in a file or part of stdin. I was able to create a file in setup method but oc binary is not able to pick it up. We can discuss it today.